duckduckgo/iOS
# 13. Iterate on Subscription Changes

Task/Issue URL: https://app.asana.com/0/0/1206362842825232/f

**Description**:
- This includes several copy and small UI changes for the iOS Subscription flow based on user feedback.   
- Memory management updates to prevent a leak on the subscriber feature and thoroughly deallocating the webviews



**Steps to test this PR**:
1.  ⚠️ No need to test for everything.   Most relevant change is the Subscription settings screen.   Make sure it displays accordingly to your current subscription with the new layout.

<img width="246" alt="Screenshot 2024-02-23 at 6 17 57 PM" src="https://github.com/duckduckgo/iOS/assets/1156669/5345081e-7f6b-478f-8f11-4045ae1efb94">


2. This PR also includes several copy changes, but no need to test them individually as those will be tested during the design review.

**When user not subscribed**
✅ update description
✅ remove “Learn more” and add “Get Privacy Pro"
✅ add row “I Have a Subscription"
When user is subscribed
✅ change “VPN (Virtual Private Network)” to “VPN"
✅ change “Subscription Settings” from link to list item with chevron

**Subscription Settings**
✅ add new hero header with Privacy Pro icon
✅ move “Change Plan or Billing” up
✅ change “Manage” to “Subscription"
✅ update FAQ description

**Add device**
✅ update subheading
✅  add email: update email box description
✅ email added: update email box description

**Remove from this device**
✅ update modal heading casing

**Activate Subscription**
✅ update subheading
✅ enter email: update description
✅ update layout to move “Restore Purchase” below
✅review success and error modals copy - check design
✅ note that the onboarding screen should not show the "add email" card for users who have an email address associated with their subscription



---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
